### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by Vishal-1705** [github/india#500] [Students]
I have submitted my request for the student developer pack, but I still did not get any response whether my request has been accepted or rejected. Please help me out. @ey-mentor1 @ey-mentor2  @EY-Mentor3  @EY-Mentor4

**Comment by Sanjana110278** [comment]
Same issue here. It says that your request has been submitted but it has neither been accepted or rejected.

**Reply by AbhishekMondal1** [reply]
You will not get dev pack from there.

**Reply by Vishal-1705** [reply]
What is the procedure then?

**Comment by ankitpartap** [comment]
I already have a student developer pack. So, will it create any problem or not?

**Comment by DevendraPhaniKumar** [comment]
**I want to discuss about two important points which is essential for all of us.**

**1.How to Claim Student developer pack from FRT portal.I tried it by clicking on Student Developer Pack which lies on the Right side of web page (There is no feedback on it such as Accepted or Rejected.....Refer Image-4)**

**2.On surfing a lot I navigated to GitHub Student Benefits but to claim the Microsoft Azure Benefits(Refer Image-1,2) the requirement is to create a school/work account which is not easy unless the school(College/university.......Refer Image-3)  is registered.I have no clue on how to create a school account..... I would like to know am I following a wrong approach.**

**As claiming the Student Developer pack is one of the criteria in tracking the progress.I would like to know the solution for it at the earliest.**

@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4
  
![1](https://user-images.githubusercontent.com/17966329/139542459-6ee99b9f-4a2e-4333-9e41-7ea90dc2ecd0.PNG)
![2](https://user-images.githubusercontent.com/17966329/139542460-b8c26a95-6614-4907-a800-7e93609b5fd8.PNG)
![3](https://user-images.githubusercontent.com/17966329/139542461-32304cfc-68c9-40b6-b160-7c6c06037d93.PNG)
![4](https://user-images.githubusercontent.com/17966329/139542463-6be9ce7a-1eca-4bb5-8f80-54cfdbe75a38.PNG)

**Comment by DevendraPhaniKumar** [comment]
I am looking on support for how to access the exciting offers mentioned in FRT portal the link is https://futurereadytalent.in/Home/Offers. The page is not completely accessible.

Kindly guide on this at your earliest 
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4
![5](https://user-images.githubusercontent.com/17966329/139543041-a0f3629e-f5b5-47f8-a5cd-e8a6f6d434cb.PNG)
